### DISCUSSION TITLE
extruder goes to X0Y0 when starting with my 3d Chameleon

### DISCUSSION THREAD

**Discussion by dpierce74** [Jyers/Marlin#1416] [Q&A] [answered]
so I have been trying to get my 3d Chameleon going, I updated the firmware to Jyers UI. Is there something in the firmware to send the extruder to X0Y0 for any reason? Ive looked in my gcode and it doesnt tell it to do this. Ive been dealing with the maker of 3d Cameleon and hes saying its the firmware. does anyone else use the 3d Chameleon? 

the GCODE tells it to go to a specific spot, it goes there but then it returns to X0Y0 and does the actions it should have done at the given specific spot. it goes to that spot to trigger a switch.  it seems to somehow override the gcode command of where to go.

**Comment by Jyers** [comment, marked as answer]
Try resetting EEPROM (Control menu, reset to defaults then store settings) 

If thats not the issue it's probably a slicer issue. Either origin at center or some other strange setting is enabled that shouldn't be
